src/renderer/screens/Preferences/keyboard/leds/DefaultLedMode.js

```diff
@@ -30,61 +30,43 @@ const DefaultLedMode = (props) => {
   const { onSaveChanges } = props;
 
   const [ledModeDefault, setLedModeDefault] = useState(0);
-  const [ledModeAutoSave, setLedModeAutoSave] = useState(true);
 
   const initialize = async (focus) => {
     const def = await focus.command("led_mode.default");
-    const autoSave = await focus.command("led_mode.auto_save");
 
     setLedModeDefault(parseInt(def));
-    setLedModeAutoSave(autoSave == "1");
   };
   const loaded = usePluginEffect(initialize);
 
-  const onAutoSaveChange = async (event) => {
-    const autoSave = event.target.checked;
-    await setLedModeAutoSave(autoSave);
-    await onSaveChanges("led_mode.auto_save", autoSave ? 1 : 0);
-  };
-
   const onLedModeChange = async (event) => {
     const v = event.target.value;
-    const mode = Math.max(0, Math.min(32, v == "" ? 0 : parseInt(v)));
+    const mode = Math.max(0, Math.min(126, v == "" ? 0 : parseInt(v)));
```

Comment:
What is the 126 here? Can you break it out to a named constant?

Reply:
That's the highest led mode we can sensibly set, and it's wrong, should be 255.

I set it to 126, because I was still in the PersistentLEDMode mindset, where we had auto-save & led mode on a single bight, so led mode was limited to 0-127, but 127 + autosave was 0xff, thus, uninitialized, so 0-126 was the range that setup supported. With DefaultLEDModeConfig, no such limitation exists, so it really should be 255.

Do we want a named constant for that?

Reply:
yeah. a constant is still worth it.